docs/reference/mapping/types/runtime.asciidoc

```diff
@@ -0,0 +1,585 @@
+[[runtime]]
+=== Runtime
+Typically, you must index fields to {es} before they can be retrieved,
+aggregated, or searched. With _runtime fields_ beta[], you can explicitly
+define a field in the mapping and access it at search time without indexing
+your data during ingest time.
+
+Because runtime fields aren't indexed, you can more quickly ingest raw data
+into the Elastic Stack and access it immediately. Runtime fields are accessible
+from the search API like any other field. The <<search-field-caps,field capabilities API>> returns runtime fields as both searchable and aggregatable.
+
+By dynamically evaluating runtime fields at search time, you can optimize disk
+space by choosing which fields to index. If you later decide that a runtime
+field is increasingly used for filtering or aggregation, you can add the field
+to `_source` to gain faster search speed.
+
+Alternatively, you can specify a `runtime_mappings` section in a search request
+to use runtime fields that exist only as part of the query. This
+flexibility allows you to create or override field values in `_source` for the
+duration of the query without modifying the field itself.
+
+Runtime fields help to alleviate several common issues when using {es}:
```

Comment:
I dunno if I'm normal but I feel like this is too much words without enough examples or headings. I don't read good on the best of days and I'm usually skimming pages for useful headings.

Reply:
This is a great observation, and is forcing me to rethink the structure of this information. I'm going to break it up into much more discrete sections that are move obvious and digestible.